.github/workflows/add_opened_pr_to_project.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       # check out the repo to get the script downloaded
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
This is good, of course.  Any preference to using the full SHA instead of a tag?  Is that safer/preferred?

Reply:
Github official actions are safe to pin to a major tag.